EDITING.md

```diff
@@ -0,0 +1,177 @@
+
+### linking
+
+The preferred linking format depends on what you're linking to and how far away it is.
+In general, the aim is to make editing easier in the long run.
+
+- The preferred link syntax depends on how distant the target is.
+  - Links within a document should use bare hashtags -- `[text](#heading)`.
+  - Links to documents in the same folder should use relative paths -- `[text](./file.md#heading)`.
+  - Links to documents elsewhere in the repo should use rooted paths -- `[text](/full/path.md#heading)`.
+    (This makes repo-wide reorganizations and searching easier; a series of "../../.." can be inconvenient to work with, and are difficult to verify by eye.)
+  - Links to content outside the repo should of course use the full URL.
+- Links should include the file extension for '.md' files.
+  (It will be stripped by our site publish tooling, but is necessary for links to work well within github's markdown processing.)
+- When linking to source code: use the full commit hash!  (Github has a handy `'y'` shortcut for this; use it!)
+  - If linking to specific line numbers, a full commit hash is an absolute requirement.  It is otherwise far too easy to have "working" but semantically invalid links.
+  - If aiming to whole packages or high level documents in other repos, it _may_ be viable to link to the master branch.  But be judicious.
+
+### todos
+
+Some documents have "TODO"s within them.
+
+This isn't necessarily something to emulate when submitting new changes.
+Documentation that's done is much better than documentation that's to-do!
+
+Whether or not a "TODO" is acceptable in a PR depends:
+
+- How critical is it?
+  - if it's a "TODO" in critical parts of system definition?  No.  (Consider re-writing what you're working on as an [exploration report](#use-exploration-reports) instead!)
+  - If it's a "TODO" stating that better description is still wanted?  Maybe.  If it's useful in context to state that, and a legitimate open ask to the community at large when they read the document, we can consider merging it.
+- How sure are we it can be addressed?
+  - If it's coming from someone with known long-term investment in the project?  Maybe; we can have a reasonable expectation the same person will either come back to finish it, or at least has a good idea about whether or not it will be possible for other contributors to fill the missing piece.
+  - If it's coming from someone making their first contributions?  Unfortunately, we have to set a higher bar here.  It's harder to know whether follow-up will come, and so its more risky to bet that it will.
+
+None of these factors have a clear yes-or-no answer (nor are they intended to),
+so, ultimately, expect this to be resolved by discussion during review.
+By default?  Fewer "TODO"s are better: avoid introducing them if you can.
+
+### line breaks
+
+Linebreaks are non-semantic in markdown, so where to break lines is up to you.
+We don't enforce any strict line-length limit.
```

Comment:
for now ... I like the vicinity rule but I think this is an area we should work on because we have something approaching "line break by feel" in this repo.

Reply:
I personally use breaks just for paragraphs, my softwrap in my editor is good enough to deal with that.